CONTACTING-OPS.md

```diff
@@ -8,7 +8,25 @@ If there is an emergency -- production is down right before a big demo, an attac
 1. If you don't have an account on the RtF Slack, skip the next few steps.
 1. Go to [#ops in Slack](https://raisingthefloor.slack.com/messages/ops/). Ask for help using `@here`. [More on Slack announcements](https://get.slack.help/hc/en-us/articles/202009646-Make-an-announcement).
 1. Go to [#ops in Slack](https://raisingthefloor.slack.com/messages/ops/). Ask for help using `@channel`.
-1. Email `ops at raisingthefloor dot org`.
+1. Email `ops at raisingthefloor dot org`. Please, attach as much detail of the issue as you can. A good template for such email could be:
+
+```
+Subject: [Type here a short description of the issue]
+Body of the message:
+
+* Location of the PC that has issue
```

Comment:
```
* Physical location of the PC that is having a problem (e.g. Pilot site in Washington, D.C.; my development laptop on my local cafe's internet)?

* Describe the problem. What is happening?
** Please include any screenshot, detailed error messages, or other details that you think might be helpful.

* What did you expect to happen instead?

* If possible, what happens use a browser on the affected PC and go to https://flowmanager.prd.gcp.gpii.net/health. What does it say?

* If possible, try to reproduce the problem on a nearby computer (e.g. another device in the same lab). Repeat the above check using the nearby computer's browser. Is anything different?
```

Reply:
Thanks @mrtyler , applying now and shipping it